src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -857,7 +861,7 @@ export async function hasDebugPortChanged(projectInfo: ProjectInfo): Promise<boo
 
     const containerName = await getContainerName(projectInfo);
     if (process.env.IN_K8 === "true") {
-        // do nothing for now, since Kubernetes does not support debug mode
+        // We do not change the port of Kubernetes projects
```

Comment:
why not? the user can change their chart to expose a new port and update .cw-settings for Kubernetes as well?

Reply:
I wasn't sure what to update this comment to, so that was just my guess. What do you think it should be?

Also, `hasDebugPortChanged` is only called during `restartProject`, so do we really support this
>the user can change their chart to expose a new port and update .cw-settings for Kubernetes as well
? We are still always returning `false` for Kube projects

Reply:
let me get back to you on this